node/Cargo.toml

```diff
@@ -103,7 +103,7 @@ prometheus-endpoint = { package = "substrate-prometheus-endpoint", git = "https:
 beefy-gadget-rpc = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
 mmr-rpc = { git = "https://github.com/sora-xor/substrate.git", branch = "polkadot-v0.9.38" }
 mmr-gadget = { git = "https://github.com/paritytech/substrate", branch = "polkadot-v0.9.38" }
-leaf-provider-rpc = { git = "https://github.com/sora-xor/sora2-common", branch = "polkadot-v0.9.38" } # 'develop' branch after PR 14
+leaf-provider-rpc = { git = "https://github.com/sora-xor/sora2-common", branch = "develop" } 
```

Comment:
Please change `https://github.com/sora-xor/sora2-common` to `https://github.com/sora-xor/sora2-common.git` in all places

Reply:
done